app/metrics.yaml

```diff
@@ -0,0 +1,75 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+nav:
+  browser_menu:
+    type: string
+    description: >
+      String of the item clicked in the menu which includes: Back, Forward, Reload, Exit Firefox
+    send_in_pings:
+      - baseline
+    lifetime: user
+    bugs:
+      - N/A
+    data_reviews:
+      - N/A
+    notification_emails:
+      - katglazko@gmail.com
```

Comment:
Notification emails should be the team list (so everyone sees the expiring glean probes, rather than a single person being responsible).